components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -72,6 +72,12 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
   template<typename T>
   using str_map = std::map<std::string,T>;
 
+  // Default base constructor
+  AtmosphereProcess () = default;
```

Comment:
Question: do we need/want a default ctor? Who uses it?

Reply:
No one currently. My only thought was if a process (or test) doesn't care about params and comms then they would only have to define their own constructor. But maybe this is not something that would happen. I can remove it.

Reply:
Uhm, I think devs of such an atm proc would not have a hard time creating dummy comm/params. OTOH, if someone accidentally uses the def ctor, there might be issues finding the bug. I think we can remove it, and if need be, add it back later.